sequencer/src/lib.rs

```diff
@@ -583,8 +564,13 @@ mod test {
 
             // Check that each successive header satisfies invariants relative to its parent: all
             // the fields which should be monotonic are.
-            for leaf in leaf_chain.iter().rev() {
+            'inner: for (i, leaf) in leaf_chain.iter().rev().enumerate() {
                 let header = leaf.block_header.clone();
+                if i == 0 {
+                    parent = header;
+                    continue 'inner;
```

Comment:
Is the `'inner` label necessary? Wouldn't just `continue;` work here?

Reply:
Yes I thought explicitness might be nice there. But it is unneeded so removed in d55614106.